setup.py

```diff
@@ -1,25 +1,84 @@
+# -*- coding: utf-8 -*-
+#
+# setup.py
+#
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+AWS SAM Serverless Application Model
+"""
+import io
+import os
+import re
 
 from setuptools import setup, find_packages
 
 
-with open('README.md') as f:
-    readme = f.read()
+def read(*filenames, **kwargs):
+    encoding = kwargs.get('encoding', 'utf-8')
+    sep = kwargs.get('sep', os.linesep)
+    buf = []
+    for filename in filenames:
+        with io.open(filename, encoding=encoding) as f:
+            buf.append(f.read())
+    return sep.join(buf)
+
+
+def read_version():
+    content = read(os.path.join(
+        os.path.dirname(__file__), 'samtranslator', '__init__.py'))
+    return re.search(r"__version__ = '([^']+)'", content).group(1)
+
+
+def read_requirements(req='base.txt'):
+    content = read(os.path.join('requirements', req))
+    return [line for line in content.split(os.linesep)
+            if not line.strip().startswith('#')]
 
-with open('LICENSE') as f:
-    license = f.read()
 
 setup(
     name='aws-sam-translator',
-    version='1.4.0',
+    version=read_version(),
     description='AWS SAM Translator is a library that transform SAM templates into AWS CloudFormation templates',
-    long_description=readme,
+    long_description=read('README.md'),
     author='Amazon Web Services',
     author_email='aws-sam-developers@amazon.com',
     url='https://github.com/awslabs/serverless-application-model',
-    license=license,
+    license='Apache Software License',
     # Exclude all but the code folders
-    packages=find_packages(exclude=('tests', 'docs', 'examples', 'versions')),
+    packages=find_packages(exclude=('docs', 'examples', 'versions')),
+    package_data={'': '*.yaml'},
+    install_requires=read_requirements('base.txt'),
+    extras_require={
+        'dev': read_requirements('dev.txt')
+    },
     keywords="AWS SAM Serverless Application Model",
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Environment :: Other Environment',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Information Technology',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 3',
```

Comment:
I think we should remove this. I don't think we are 3 compliant yet, let's remove this until we fully support Python3.